lib/receive_intent_manager.dart

```diff
@@ -32,8 +32,6 @@ class ReceiveIntentManager {
 
     receiveMediaWhenInMemory();
```

Comment:
How does this even work? The user has to say on the root screen's onLoad that he/she wants to listen to the intent?

Reply:
It's working even if I again adding the `receiveIntent` action in the next page's `onLoad` action and able to get the datas